package.json

```diff
@@ -27,22 +27,24 @@
     "broccoli-asset-rev": "^2.4.6",
     "elemental-calendar": "0.0.9",
     "ember-ajax": "2.5.3",
-    "ember-cli": "2.10.0",
+    "ember-cli": "2.11.1",
     "ember-cli-app-version": "^2.0.0",
     "ember-cli-clipboard": "0.5.0",
     "ember-cli-dependency-checker": "^1.3.0",
     "ember-cli-eslint": "3.0.2",
-    "ember-cli-htmlbars-inline-precompile": "^0.3.5",
+    "ember-cli-htmlbars": "^1.1.1",
```

Comment:
htmlbars is in the `dependencies` key here.  I think that is where it is supposed to go in order to render templates that get passed up the chain.  Some investigation may be in order, but it for sure doesn't need to be in both locations.